src/Analysis/Ast/Impl/Analyzer/PythonAnalyzerEntry.cs

```diff
@@ -93,7 +94,7 @@ internal sealed class PythonAnalyzerEntry {
         }
 
         public Task<IDocumentAnalysis> GetAnalysisAsync(CancellationToken cancellationToken)
-            => _analysisTcs.Task.ContinueWith(t => t.GetAwaiter().GetResult(), cancellationToken);
+            => _analysisTcs.Task.WaitAsync(cancellationToken);
```

Comment:
@AlexanderSher ?

Reply:
Also from #1596, so we should probably review that first.